dataprofiler/labelers/base_model.py

```diff
@@ -366,10 +371,10 @@ def fit(
         train_data: Union[pd.DataFrame, pd.Series, np.ndarray],
         val_data: Union[pd.DataFrame, pd.Series, np.ndarray],
         batch_size: int = 32,
-        epochs: int = 1,
         label_mapping: Dict[str, int] = None,
         reset_weights: bool = False,
-    ) -> None:
+        verbose: bool = True,
+    ) -> Tuple[Dict, Optional[float], Dict]:
```

Comment:
why is `verbose` being added and `epochs` being removed?

Reply:
does mypy complain about that?

Reply:
mypy complains that `Signature of "fit" incompatible with supertype "BaseTrainableModel"` for all of the subclasses